include/varcache.h

```diff
@@ -1,5 +1,50 @@
 #include <usual/strpool.h>
 
+#undef HASH_FUNCTION
+#define HASH_FUNCTION(keyptr,keylen,hashv) HASH_JEN_LOWERCASE(keyptr, keylen, hashv)
+
+#define HASH_JEN_LOWERCASE(key,keylen,hashv) \
```

Comment:
Let's put this define in a separate file: `include/common/uthash_lowercase.h`

That way we can include the original license. 

Also let's add a comment explaining that this is a copy of HASH_JEN, but with an added `tolower` call for every place where you get a character.